karaf/apache-brooklyn/src/main/resources/etc/org.apache.brooklyn.classrename.cfg

```diff
@@ -0,0 +1,21 @@
+################################################################################
```

Comment:
I believe there's some mechanism to have the files self-extract on first boot or even during assembly. Should look it up.

I agree we should make the distribution assembly re-usable, not only files in etc.